main/gnutls/APKBUILD

```diff
@@ -2,12 +2,13 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gnutls
-pkgver=3.6.4
+pkgver=3.6.7
 pkgrel=0
 pkgdesc="A TLS protocol implementation"
 url="https://www.gnutls.org/"
 arch="all"
 license="GPL"
```

Comment:
Should be changed to `LGPL-2.1-or-later`
https://gitlab.com/gnutls/gnutls/blob/master/LICENSE